src/art.rs

```diff
@@ -1065,7 +1065,7 @@ impl<P: KeyTrait, V: Clone> Tree<P, V> {
         self.insert_common(key, value, version, ts, true)
     }
 
-    pub fn insert_without_version_increment_check(
+    pub fn insert_unchecked(
```

Comment:
I think the suffix `_unchecked()` is a bit confusing, because the standard library effectively reserves it for unsafe methods.

Reply:
Thanks @gsserge, any suggestion on what could be a better alternative?

Reply:
Yeah, finding a good name for this one is tricky and I can see how `insert_unchecked()` is a viable option.

I think `insert_without_version_check()` is a good balance between being descriptive and not too long, but if you still think it's way to long then `insert_unchecked()` is also good.

Reply:
I see your point. The reason I wanted to keep it short was to avoid it being tied to anything specific, so that if we later want to add some checks, the name doesn't have to be changed. However, I'll keep this open in case we do figure out a better name :)